linkerd/app/src/env.rs

```diff
@@ -187,11 +187,29 @@ pub const ENV_INBOUND_IPS: &str = "LINKERD2_PROXY_INBOUND_IPS";
 pub const ENV_IDENTITY_DISABLED: &str = "LINKERD2_PROXY_IDENTITY_DISABLED";
 pub const ENV_IDENTITY_DIR: &str = "LINKERD2_PROXY_IDENTITY_DIR";
 pub const ENV_IDENTITY_TRUST_ANCHORS: &str = "LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS";
-pub const ENV_IDENTITY_IDENTITY_LOCAL_NAME: &str = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME";
 pub const ENV_IDENTITY_TOKEN_FILE: &str = "LINKERD2_PROXY_IDENTITY_TOKEN_FILE";
 pub const ENV_IDENTITY_MIN_REFRESH: &str = "LINKERD2_PROXY_IDENTITY_MIN_REFRESH";
 pub const ENV_IDENTITY_MAX_REFRESH: &str = "LINKERD2_PROXY_IDENTITY_MAX_REFRESH";
 
+// This config is here for backwards compatibility. If set, both the tls id and the server
+// name will be set to the value specified in this config. The values needs to be a DNS
+// name
```

Comment:
```suggestion
/// This config is here for backwards compatibility. If set, both the tls id and the server
/// name will be set to the value specified in this config. The values needs to be a DNS
/// name
```

Definitely a nit, but opt to use doc-comments -- especially when pub.